doc/bakugo_spec.md

```diff
@@ -414,22 +414,22 @@ rune        alias for int32
 
 ```
 Signature      = Parameters [ Result ] .
-Result         = Parameters | Type .
+Result         = Type .
```

Comment:
```suggestion
Result         = "(" [ ResultList [ "," ] ] ")" | Type .
ResultList     = Type { "," Type } .
```
If I am not wrong, production `Result = Type .` will not allow the declaration of multiple return values.

The production name `ResultList` does not make much sense. I could not come up with a better one though.

Reply:
Oh you're right.

> The production name ResultList does not make much sense. I could not come up with a better one though.

How about not making it a separate non-terminal?

```suggestion
Result         = "(" [ Type { "," Type } [ "," ] ] ")" | Type .
```

Reply:
Okay. It feels a bit crowded, but that will do.